x-pack/plugins/serverless_search/public/application/components/connectors_overview.tsx

```diff
@@ -112,6 +123,29 @@ export const ConnectorsOverview = () => {
       </EuiPageTemplate.Header>
       {connectorsLoading || (data?.connectors || []).length > 0 ? (
         <EuiPageTemplate.Section restrictWidth color="subdued">
+          <EuiCallOut
+            size="m"
+            title="Coming soon Elastic managed connectors."
+            iconType="pin"
+            onDismiss={() => {}}
+          >
+            <p>
+              {i18n.translate('xpack.serverlessSearch.connectorsOverview.calloutDescription', {
+                defaultMessage:
+                  'We´re actively developing Elastic managed connectors, that won´t require any self-managed infrastructure. You´ﬁll be able to handle all configuration in the UI. This will simplify syncing your data into a serverless Elasticsearch project. ',
```

Comment:
These apostrophes should be apostrophes, not backticks. Also there's something weird going on at the start of the second sentence.